catalogs/catalog-hive/build.gradle.kts

```diff
@@ -129,6 +129,10 @@ dependencies {
   testImplementation(libs.testcontainers.mysql)
   testImplementation(libs.testcontainers.localstack)
   testImplementation(libs.hadoop2.aws)
+  testImplementation(libs.hadoop3.abs)
+
+  // You need this to run test CatalogHiveABSIT
+  // testImplementation(libs.hadoop3.common)
```

Comment:
Is this comment still valid?

Reply:
Explain the reason why this dependency should be comment out, not just adding the confusing comment makes reviewer hard to understand.